include/aws/s3/private/s3_meta_request_impl.h

```diff
@@ -184,9 +185,6 @@ struct aws_s3_meta_request {
         /* Note: this needs to be first for using AWS_CONTAINER_OF with the nested structure. */
         struct aws_linked_list_node node;
 
-        /* True if this meta request is currently in the client's list. */
-        bool scheduled;
```

Comment:
I'm not sure what this was ever used for

We have branches that handle a meta-request being re-added to the client, but that's not possible. I was touching code near those branches and was like "how do I handle this ... thing that can't actually happen"

so instead of thinking how I'd handle an impossible bit of state, I removed the state.